src/VisualStudio/CSharp/Impl/CodeModel/CSharpCodeModelService.cs

```diff
@@ -904,13 +907,19 @@ public override SyntaxNode SetName(SyntaxNode node, string name)
                 case SyntaxKind.Parameter:
                     return ((ParameterSyntax)node).WithIdentifier(newIdentifier);
                 case SyntaxKind.NamespaceDeclaration:
-                    return ((NamespaceDeclarationSyntax)node).WithName(SyntaxFactory.IdentifierName(name));
+                    return ((NamespaceDeclarationSyntax)node).WithName(
+                        SyntaxFactory.ParseName(name)
+                            .WithLeadingTrivia(SyntaxFactory.TriviaList(SyntaxFactory.ElasticMarker))
```

Comment:
consider extracting out a readonly static for the elastic trivia list.

Reply:
I would not expect extracting it to make any measurable difference. `SyntaxTriviaList` is a struct and this construction pattern is allocation free and very cheap.

Reply:
rather than doing this pattern repeatedly and other could get into this issue again, how about creating helper that does this and use that through code?

Reply:
I wasn't suggesting it for perf.  But for reducing duplication.